### PR TITLE
Publish to buildkite

### PR DESCRIPTION
**Pull request by itsmichaelwang**

### CODE REVIEW

**Comment by itsmichaelwang** [comment]
Just to be safe, I'll wait until the code is all migration to `@gusto/dom-to-image` before merging this in.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+steps:
+  - name: ':npm: Publish to npm'
+    command: '.buildkite/run.sh .buildkite/publish.js'
+    agents:
+      queue: npm-publish
+  - name: ':book: Deploy documentation'
```
Comment (jeffreykwan) [inline .buildkite/pipeline.yml]: whoops my bad, ths step `:book: Deploy documentaiton` is not necessary

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+steps:
+  - name: ':npm: Publish to npm'
+    command: '.buildkite/run.sh .buildkite/publish.js'
```
Comment (jeffreykwan) [inline .buildkite/pipeline.yml:3]: let's add branches: 'master' to prevent accidental publishing

**Comment by itsmichaelwang** [comment]
Also, I added back in the code that checks for duplicate versions and skips publishing if a version already exists.

**Review by jeffreykwan — APPROVED** [review]

**Comment by zp-alfred** [comment]
Accepted by jeffreykwan!

**Pull request merged**